dokka-subprojects/plugin-base-frontend/webpack.config-ui-kit.js

```diff
@@ -66,12 +88,26 @@ module.exports = (env, args) => {
           ],
           exclude: /node_modules/,
         },
+        {
+          test: /\.(png|jpe?g|gif|svg)$/i,
+          type: 'asset/resource',
+        },
       ],
     },
     plugins: [
       new MiniCssExtractPlugin({
         filename: isMinify ? 'ui-kit.min.css' : 'ui-kit.css',
       }),
+      new WebpackShellPluginNext({
+        onDoneWatch: {
+          scripts: [
+            'echo "Done rebuild, coping files to dokka-integration-tests"',
+            'cp -r ../plugin-base/src/main/resources/dokka/ui-kit ../../dokka-integration-tests/gradle/build/ui-showcase-result',
```

Comment:
and this copies rewritten ui-kit's output to the ui-showcase directory so the changes are reflected immediately